tests/proxy/ui.t

```diff
@@ -0,0 +1,50 @@
+  C\x00\x00\x00\x00\x00\x00\x00\x00\x00\x14L\t\x11K<h\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00_S\x1di\x139M\x00\x00\x00\x00\x00\x00\x00\x00\x0072clY\x1f\x0f\x00\x00\x00\x00\x00\x00\x00\x00\x00AE! W\x19k\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x0bZaU\x01b\x18\x00\x00\x00\x00\x00\x00\x00\x00\x00\x1c5\x1aV\x023\x06\x00\x00\x00\x00\x00\x00\x00\x00\x00D0OR4[\x15\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x0e:\x04B@\x07+\x00\x00\x00\x00\x00\x00\x00\x00\x00(`f\x1enN\x08\x00\x00\x00\x00\x00\x00\x00\x00\x00/d-\x0c\r (no-eol) (esc)
+  8Q\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'^),#\x16\x00\x00\x00\x00\x00\x00\x00\x00\x00\x001mX6?$\x05\x00\x00\x00\x00\x00\x00\x00\x00\x00].jGT&>\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00e;=\x17\\F\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x12I\x1bJH*%\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00 (no-eol) (esc)
+  $ wget http://127.0.0.1:8002/a/repo && cat repo
+  Connecting to 127.0.0.1:8002 (127.0.0.1:8002)
+  saving to 'repo'
+  repo                 100% |********************************|   633  0:00:00 ETA
```

Comment:
wget progress bar is probably not very useful in those tests; there could be an option to disable it, or just use curl